docker-compose.yaml

```diff
@@ -0,0 +1,24 @@
+slackbot:
+  image: node:6
+  entrypoint: /app/entrypoint.sh
+  environment:
+    - DATABASE_URL=postgres://postgres:standup@database/standup
+    - PORT=3000
+    - LOG_LEVEL=10
+  expose:
+    - 3000
+  ports:
+    - 3000
```

Comment:
The `ports` line [binds a container port to a host port](https://docs.docker.com/compose/compose-file/#/ports).  If you don't specify the host port, it uses a random one, which makes it a hassle to find.  Instead, I think we should specify both, so we always know what port to look for on the host?

Also, we could use environment interpolation so we don't have the same hardcoded value in multiple places.  So, for example, maybe the `ports` line would end up being something like this:

```
ports:
  - 3000:${PORT}
```

The host port would be 3000, and the container port would be whatever was setup in the `environment`.